pynws/simple_nws.py

```diff
@@ -300,6 +305,11 @@ def forecast_hourly(self):
         """Return forecast hourly."""
         return self._convert_forecast(self._forecast_hourly, self.filter_forecast)
 
+    @property
+    def detailed_forecast(self):
```

Comment:
Should we also add a method to give the hourly forecast from the details?

Reply:
Here's how I'm using it.
```
    nws = SimpleNWS(*COORD, USERID, session)
    await nws.set_station()
    await nws.update_detailed_forecast()

    start_time = datetime.now(tz=timezone.utc) + timedelta(hours=1)
    details = list(nws.detailed_forecast.get_details_by_hour(start_time, 24))
```

Reply:
Makes sense to me.  No need to have it here explicitly!